apps/frontend/src/pages/access-management/access-management.tsx

```diff
@@ -4,29 +4,39 @@ import {
 	useAppSelector,
 	useEffect,
 } from "~/libs/hooks/hooks.js";
+import { actions as groupActions } from "~/modules/groups/groups.js";
 import { actions as userActions } from "~/modules/users/users.js";
 
+import { getGroupColumns } from "./libs/helpers/get-group-columns.helper.js";
+import { getGroupRows } from "./libs/helpers/get-group-rows.helper.js";
 import { getUserColumns } from "./libs/helpers/get-user-columns.helper.js";
 import { getUserRows } from "./libs/helpers/helpers.js";
-import { type UserRow } from "./libs/types/types.js";
+import { type GroupRow, type UserRow } from "./libs/types/types.js";
 import styles from "./styles.module.css";
 
 const AccessManagement = (): JSX.Element => {
 	const dispatch = useAppDispatch();
 	const users = useAppSelector(({ users }) => users.users);
+	const groups = useAppSelector(({ groups }) => groups.groups);
 
 	useEffect(() => {
 		void dispatch(userActions.loadAll());
+		void dispatch(groupActions.loadAll());
 	}, [dispatch]);
 
-	const columns = getUserColumns();
-	const data: UserRow[] = getUserRows(users);
+	const userColumns = getUserColumns();
+	const userData: UserRow[] = getUserRows(users);
+
+	const groupColumns = getGroupColumns();
+	const groupData: GroupRow[] = getGroupRows(groups);
 
 	return (
 		<PageLayout>
 			<p className={styles["title"]}>Access Management</p>
 			<p className={styles["sub-title"]}>Users</p>
```

Comment:
Haven't noticed this previously, let's use appropriate tags and wrap each section with `section`
```suggestion
			<h1 className={styles["title"]}>Access Management</p>
			<section>
			  <h2 className={styles["section-title"]}>Users</p>
			  //...table
			<section>
```